config/manifest/eksa-components.yaml

```diff
@@ -3806,6 +3806,75 @@ spec:
                   name:
                     type: string
                 type: object
+              packages:
```

Comment:
qq: Why do we need this?

Reply:
This was added when I ran `make generate-manifests`, so it must've been added recently and we just never generated the new manifest then